package.json

```diff
@@ -86,7 +86,7 @@
     "auth0-extension-tools": "1.4.0",
     "auth0-extension-ui": "^1.0.1",
     "auth0-oauth2-express": "^1.1.8",
-    "auth0-source-control-extension-tools": "3.5.1",
+    "auth0-source-control-extension-tools": "dev",
```

Comment:
Don't forget to update this version after releasing `auth0-source-control-extension-tools` with hooks support. And also change version in `externals` above.